### DISCUSSION TITLE
Is it possible to set up multiple notification channels for separate notification levels?

### DISCUSSION THREAD

**Discussion by kjenneycloudhealth** [containrrr/watchtower#698] [Q&A] [answered]
So I have a single watchtower container running that is monitoring containers on my system. I'd like to send error messages from this watchtower container to a specific slack channel. I'd like to also send debug messages from this specific container to a separate channel for debugging purposes. Is this currently possible>

**Comment by piksel** [comment, marked as answer]
No, there is no support for that unfortunately. Perhaps you could use the IFTT integration as a broker to get it to work?
